src/test/java/com/shopify/ShopifySdkTest.java

```diff
@@ -303,28 +304,75 @@ public void givenSomeShopifyFulfillmenmtCreationRequestWhenCreatingShopifyFulfil
 
 	}
 
-	@Test
-	public void givenSomeShopifyFulfillmenmtCreationRequestWhenCreatingShopifyFulfillmentThenCreateAndReturnFulfillmentWithFulfillmentOrderApi()
-			throws JsonProcessingException, ConnectException {
+	@Test(expected = ShopifyUnsupportedActionException.class)
+	public void givenSomeShopifyFulfillmentOrderWithNoCreateFulfillmentSupportedActionWhenCreatingShopifyFulfillmentThenGetAnUnsupportedActionException()
+			throws JsonProcessingException, ConnectException, ShopifyUnsupportedActionException {
+		final String lineItemId = "987";
+		final String fulfillmentOrderId = "1234";
 
 		final ShopifyLineItem lineItem = new ShopifyLineItem();
-		lineItem.setId("some_line_item_id");
+		lineItem.setId(lineItemId);
 		lineItem.setSku("some_sku");
 		lineItem.setQuantity(5L);
 
+		List<ShopifyFulfillmentOrderLineItem> fulfillmentOrderLineItems = new LinkedList<>();
+		ShopifyFulfillmentOrderLineItem fulfillmentOrderLineItem = new ShopifyFulfillmentOrderLineItem();
+		fulfillmentOrderLineItem.setQuantity(1);
+		fulfillmentOrderLineItem.setLineItemId(lineItemId);
+		fulfillmentOrderLineItem.setFulfillableQuantity(1);
+		fulfillmentOrderLineItem.setFulfillmentOrderId(fulfillmentOrderId);
+		fulfillmentOrderLineItems.add(fulfillmentOrderLineItem);
+
+		final List<String> supportedActions = new LinkedList<>();
+		supportedActions.add("move");
+
+		final ShopifyFulfillmentOrder fulfillmentOrder = new ShopifyFulfillmentOrder();
+		fulfillmentOrder.setId(fulfillmentOrderId);
+		fulfillmentOrder.setLineItems(fulfillmentOrderLineItems);
```

Comment:
Can you add a test here to show the case where fulfillmentOrder lineitems list is empty? I don't see a test here for that?

Reply:
yep, there isn't one
the one I added acts as one because the fulfillmentOrder has no supported actions so in the end the result will be the same
but it'll be good to add one more with the conditions reversed

Reply:
Yea, it'd be good for regression to have both cases covered for if not anything the regression we get. So future changes don't break what we expect is happening.